cpp/tests/strings/contains_tests.cpp

```diff
@@ -354,6 +355,57 @@ TEST_F(StringsContainsTests, CountTest)
   }
 }
 
+TEST_F(StringsContainsTests, FixedQuantifier)
+{
+  auto input = cudf::test::strings_column_wrapper({"a", "aa", "aaa", "aaaa", "aaaaa", "aaaaaa"});
+  auto sv    = cudf::strings_column_view(input);
+
+  {
+    // exact match
+    auto results = cudf::strings::count_re(sv, "a{3}");
+    cudf::test::fixed_width_column_wrapper<int32_t> expected({0, 0, 1, 1, 1, 2});
```

Comment:
I guess there are two discussions here:
1. Is this behavior implementation-defined, or is it standard? If it's standard, then it doesn't need to be documented at all and my question can be chalked up to my own ignorance/forgetfulness, but if it's implementation-defined then we need to document it somewhere.
2. If it _does_ need to be documented, should that be done in this PR? I'm fine with you doing that in a separate (nice and tiny) PR. I just figured that the end result would be that the fix got squeezed into another regex PR, but a standalone one is great too if you want to do that.

Reply:
It's pretty standard I think. I don't mind adding more documentation but I'm not sure it belongs in this PR. I can be persuaded since this is a _cleanup_ PR.

```
>>> import pandas as pd
>>> ps = pd.Series(["a", "aa", "aaa", "aaaa", "aaaaa", "aaaaaa"])
>>> ps.str.count("a{3}")
0    0
1    0
2    1
3    1
4    1
5    2

```

Reply:
If this is how other implementations behave as well I don't think we need documentation to call attention to it, I just wanted to make sure. Thanks for verifying.